systemtest/src/main/java/io/strimzi/systemtest/enums/ResourceOperation.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.enums;
+
+public enum ResourceOperation {
+    RESOURCE_CREATION,
+    RESOURCE_ROLLING_UPDATE,
+    RESOURCE_READY
```

Comment:
How do you recognize which resource you want to create (topic, pod, deployment, kafka, etc) ?

Reply:
I have it in the method with `shortTimeout` variable name..